src/marqo/s2_inference/hf_utils.py

```diff
@@ -2,10 +2,18 @@
 from torch import nn
 from transformers import (AutoModel, AutoTokenizer)
 import numpy as np
-
```

Comment:
bad ordering and splitting of imports. Should be alphabetical and split: standard/packages/local

Reply:
Thanks for the advice here! I didn't even notice before!

Reply:
resolved